index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Personal Portfolio Website</title>
+</head>
+
+<body>
+    <header>
+        <nav class="navigation">
+            <div class="logo">
+                <h3>Dhuruv</h3>
+            </div>
+            <div class="menu">
+
+            </div>
+        </nav>
+    </header>
+    <main>
+        <div class="headline-section">
+            <div class="illustration-mobile">
+                <img src="./icons/header-illsutration-mobile@2x.png" alt="">
+            </div>
+            <div class="heading-text">
+                <h1>Hey there. I’m Dhuruv</h1>
+                <h1 class="h-green">I’m a software developer</h1>
+            </div>
+            <div class="supporting-text">
+                <p> I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+            </div>
+            <ul class="social-icons">
+                <li>
+                    <a href="https://github.com/DhuruvSingh">
+                        <img src="./Social-icon-black/GitHub.png" alt="Github icon">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/dhuruv-singh-a1a51aa9/">
+                        <img src="./Social-icon-black/Linkedin.png" alt="Linkedin icon">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://angel.co/u/dhuruv-singh">
+                        <img src="./Social-icon-black/Angelist.png" alt="Angelist icon">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://twitter.com/Ritiksi76216487">
+                        <img src="./Social-icon-black/Twitter.png" alt="Twitter icon">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://medium.com/@dhuruvs096">
+                        <img src="./Social-icon-black/Medium.png" alt="Medium icon">
+                    </a>
+                </li>
+            </ul>
+        </div>
+    </main>
```

Comment:
- [ ] Please the headline section should be contained in `section` instead of the `main` tag. there should be only one `main` tag in the whole document as it is semantically better.

Reply:
okey